src/components/modules/Footer/Footer.js

```diff
@@ -37,7 +39,7 @@ export default class Footer extends React.Component {
           </Link>
         </div>
         <div className={style.OrgInfoArea}>
-          Terms of Service | Privacy Policy | ©Dev Launchers, 2020.
+          Terms of Service | Privacy Policy | ©Dev Launchers, {year}.

```

Comment:
One nit that is _not_ important — I would just inline this:

```js
           Terms of Service | Privacy Policy | ©Dev Launchers, {new Date().getFullYear()}. 
```

Reply:
I can change it - lmk.